pkg/cli/alpha/config-gen/controller-gen-filter.go

```diff
@@ -64,7 +64,10 @@ func (cgr ControllerGenFilter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error
 
 	// set the directory
 	b := bufferedGenerator{}
-	rt, _ := gens.ForRoots(cgr.Spec.CRDs.SourceDirectory) // ignore the spurious error
+	rt, err := gens.ForRoots(cgr.Spec.CRDs.SourceDirectory)
+	if err != nil {
+		return nil, errors.WrapPrefixf(err, "failed to produce generator's Runtime against %s", cgr.Spec.CRDs.SourceDirectory)
+	}
```

Comment:
Begin to return the error does not seems to be related to any go get issue.
That seems be solved only by the fact of we upgrade the controller-tools used by config-gen

I'd recommend:
- Revert this part and leave this PR just to upgrade the dep
- then we can check why the errors are ignored and if we should or not return them. That shows very intentional the reason for them be ignored.